CHANGELOG.rst

```diff
@@ -16,6 +16,12 @@ Subsections for each version can be one of the following;
 
 Each individual change should have a link to the pull request after the description of the change.
 
+1.1.2 (2024-02-7)
+------------------
+Added
+^^^^^
+- Update OneHotEncodingTransformer to default to returning int8 columns https://github.com/lvgig/tubular/pull/175
```

Comment:
please update link to same format as rest of changelog

Reply:
updated here https://github.com/lvgig/tubular/pull/175/commits/72cd91f670a7547dd1384cb3b9efa06f7cced9fb